### PR TITLE
Add get Pose2d/Pose3d to NTUtills

### PR DESCRIPTION
**Pull request by m10653**
Add methods to construct pose2d/pose3d objects from network tables. Getting trajectories are not not possible because only position and rotations are published to NT and not time.

### CODE REVIEW

File: src/main/java/org/jmhsrobotics/warcore/nt/NT4Util.java [hunk 1]
```diff
@@ -36,7 +39,7 @@ public static boolean putPose3d(String key, Pose3d... value) {
 	}
 
 	/**
-	 * Put a Pose2d array in the table
+	 * Put a Pose2d array into Network Tables
 	 *
 	 * @param key
 	 *            the key to be assigned to
```
Comment (m10653) [inline src/main/java/org/jmhsrobotics/warcore/nt/NT4Util.java:45]: Please change this line to not be bad. :P

**Review by m10653 — COMMENTED** [review]
REview

**Review by Lu-han-wang — APPROVED** [review]
Good Code!!!

**Comment by m10653** [comment]
This Closes #4

**Pull request merged**